factorio-map-process.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+
+## Defaults
```

Comment:
Is `Defaults` the right word here, when not all of these can be overridden? Not sure if they should be though...

Reply:
I think defaults is fine. All of these are configurable or derived.  If you want to make the path to the python script configurable, that could be a future improvement.